lib/ramble/ramble/util/logger.py

```diff
@@ -40,7 +41,7 @@ def add_log(self, path):
         """
         if isinstance(path, str) and self.enabled:
             stream = None
-            stream = open(path, 'a+')
+            stream = llnl.util.tty.log.Unbuffered(open(path, 'a+'))
```

Comment:
Mostly because our `logger` utility is a wrapper on the `llnl.util.tty.log`. So using `Unbuffered` here makes sure we obey the same unbuffered setting that the log utility expects.

I don't know what the difference is between `buffering=0` vs. using the log unbuffered, but I want to make sure we're using things from the llnl utility as long as we're just wrapping their behavior.